### DISCUSSION TITLE
post/ifkakao2022-batch-performance-read/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#27] [Announcements]
# post/ifkakao2022-batch-performance-read/

if(kakao)2022 대량의 데이터를 Batch로 읽을 때의 노하우를 공유합니다.

https://tech.kakaopay.com/post/ifkakao2022-batch-performance-read/

**Comment by Jonny-Cho** [comment]
참고가 많이 되었습니다. 양질의 자료 감사합니다!

**Comment by Hwan0518** [comment]
감사합니다! 너무 좋네요

**Comment by limwoobin** [comment]
도움이 많이 되었어요 감사합니다!

**Comment by goodchoi** [comment]
너무 좋은 글입니다!
